src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,41 @@
+package calculator;
+
+public class StringCalculator {
+
+    public static int calculate(String input) {
+        nullCheck(input);
+
+        String[] splitInput = input.split(" ");
+        int sum = Integer.parseInt(splitInput[0]);
+
+        for (int i = 1; i < splitInput.length; i += 2) {
+            sum = operate(splitInput[i], sum, Integer.parseInt(splitInput[i+1]));
+        }
+
+        return sum;
+    }
+
+    private static void nullCheck(String input) {
+        if (input == null) {
+            throw new IllegalArgumentException("입력 값이 공백입니다.");
+        }
+    }
+
+    private static int operate(String s, int left, int right) {
+        if(s.equals("+")){
+            return Operator.add(left,right);
+        }
```

Comment:
```suggestion
        if (s.equals("+")) {
            return Operator.add(left, right);
        }
```
띄어쓰기 컨벤션을 지켜주세요 ! 
IntelliJ를 사용하고 계시다면 Command + Option + L 을 사용할수 있습니다!